streams/de_adultswim.m3u

```diff
@@ -2,22 +2,56 @@
 #EXTINF:-1 tvg-id="AdultSwimAquaTeenHungerForce.us" status="online",Adult Swim Aqua Teen Hunger Force (1080p) [Geo-blocked]
 https://adultswim-vodlive.cdn.turner.com/live/aqua-teen/stream_de.m3u8
 #EXTINF:-1 tvg-id="AdultSwimBlackJesus.us" status="online",Adult Swim Black Jesus (1080p) [Geo-blocked]
-https://adultswim-vodlive.cdn.turner.com/live/black-jesus/stream_de.m3u8
+http://adultswim-vodlive.cdn.turner.com/live/black-jesus/stream_de.m3u8
 #EXTINF:-1 tvg-id="AdultSwimChannel5.us" status="online",Adult Swim Channel 5 (1080p) [Geo-blocked]
 https://adultswim-vodlive.cdn.turner.com/live/channel-5/stream_de.m3u8
+#EXTINF:-1 tvg-id="AdultSwimDailyLiveAction.us" status="online",Adult Swim Daily Live Action (1080p) [Geo-blocked]
+https://adultswim-vodlive.cdn.turner.com/live/daily_liveaction_1/stream_de.m3u8
+#EXTINF:-1 tvg-id="AdultSwimDailyAnimated.us" status="online",Adult Swim Daily Animated (1080p) [Geo-blocked]
+https://adultswim-vodlive.cdn.turner.com/live/daily_animated_1/stream_de.m3u8
 #EXTINF:-1 tvg-id="AdultSwimDreamCorpLLC.us" status="online",Adult Swim Dream Corp LLC (1080p) [Geo-blocked]
-https://adultswim-vodlive.cdn.turner.com/live/DREAM-CORP-LLC/stream.m3u8
+http://adultswim-vodlive.cdn.turner.com/live/DREAM-CORP-LLC/stream_de.m3u8 
+#EXTINF:-1 tvg-id="AdultSwimHomeMovies.us" status="online",Adult Swim Home Movies (1080p) [Geo-blocked]
+https://adultswim-vodlive.cdn.turner.com/live/test-2/stream_de.m3u8
```

Comment:
this same link is put as "test stream 2" below, is it the correct one ?

Reply:
test stream are random stuff  in any  time and could have same stream of other link